catalog/ftp/pom.xml

```diff
@@ -164,7 +164,8 @@
                             catalog-core-api-impl,
```

Comment:
Are we able to increase the jacoco numbers in this pom now that we have tests covering all the overridden methods?